docs/google-drive.md

```diff
@@ -0,0 +1,32 @@
+Getting the Google Drive integration working locally
+====================================================
+
+The app supports the ability to select documents (for annotation) via Google Drive. These steps allow you to enable that feature. The outcome of this process will be a configured Google project and valid values for the `GOOGLE_APP_ID`, `GOOGLE_DEVELOPER_KEY` and `GOOGLE_CLIENT_ID` environment variables.
```

Comment:
We may be able to streamline this for H staff in future by creating a shared set of credentials that they can just copy and paste.

I have started creating an `lms-dev` project in the Hypothesis organization here: https://console.developers.google.com/apis/credentials?project=lms-dev-250514&folder

(Note that we already have `lms-qa` and `lms-prod` projects used in QA/production respectively)

I copied and adjusted the settings from the setup that I personally use.